packages/foam-vscode/src/core/utils/path.ts

```diff
@@ -0,0 +1,88 @@
+import { posix } from 'path';
+import { promises, constants } from 'fs';
+
+export function isIdentifier(uriPath: string): boolean {
```

Comment:
`getShortestIdentifier` is in a similar position now, living in `core/utils/core.ts` because it's used in `wikilinks-diagnostics.ts`. So, there are a couple of identifier related utilities (`getShortestIdentifier`  and `isIdentifier`) that haven't a clear home. Having to choose between `core/utils/core.ts` and `core/utils/path.ts` I believe I prefer the second one. And I don't like the idea of putting them in different places. Alternatively, both can be made workspace static public functions.

Reply:
`getShortestIdentifier` and `isIdentifier` are now static functions of `FoamWorkspace`. If you prefer I can make them functions of the workspace module, but it's easier to just import `FoamWorkspace` and get everything there.